deploy/scripts/src/accounts/id-reuse.ts

```diff
@@ -0,0 +1,175 @@
+import { sleep, group, check, fail } from 'k6'
+import { type Options } from 'k6/options'
+import http, { type Response } from 'k6/http'
+import { Trend } from 'k6/metrics'
+import { selectProfile, type ProfileList, describeProfile } from '../common/utils/config/load-profiles'
+
+const profiles: ProfileList = {
+  smoke: {
+    idReuse: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 1,
+      stages: [
+        { target: 1, duration: '60s' } // Ramps up to target load
+      ],
+      exec: 'idReuse'
+    }
+
+  },
+  load: {
+    idReuse: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 45,
+      stages: [
+        { target: 3, duration: '15m' }, // Ramps up to target load
+        { target: 3, duration: '15m' }, // Steady State of 15 minutes at the ramp up load i.e. 3 iterations/second
+        { target: 0, duration: '5m' } // Ramp down duration of 5 minutes.
+      ],
+      exec: 'idReuse'
+    }
+
+  }
+}
+
+const loadProfile = selectProfile(profiles)
+
+export const options: Options = {
+  scenarios: loadProfile.scenarios,
+  thresholds: {
+    http_req_duration: ['p(95)<=1000', 'p(99)<=2500'], // 95th percentile response time <=1000ms, 99th percentile response time <=2500ms
+    http_req_failed: ['rate<0.05'] // Error rate <5%
+  }
+}
+
+export function setup (): void {
+  describeProfile(loadProfile)
+}
+
+const transactionDuration = new Trend('duration')
+
+const env = {
+  envURL: __ENV.ID_REUSE_URL,
+  envMock: __ENV.ID_REUSE_MOCK,
+  envApiKey: __ENV.ID_REUSE_API_KEY,
+  envApiKeySummarise: __ENV.ID_REUSE_API_KEY_SUMMARISE
```

Comment:
Rename to fit to `<POD>_<TEAM>_<VAR_NAME>` or `<POD>_<VAR_NAME>` and update `deploy/template.yaml`